### DISCUSSION TITLE
Specific Layout

### DISCUSSION THREAD

**Discussion by izanoth** [vercel/next.js#73629] [Help]
### Summary

According the **Layout and Templates documentation**

> If you were to combine the two layouts above, the root layout (app/layout.js) would wrap the dashboard layout (app/dashboard/layout.js), which would wrap route segments inside app/dashboard/*.

But it is not clear about isolated layouts on a specific route.
To be most enlighten, i have:

app
app/layout.tsx
app/page.tsx

and 

app/blog
app/blog/layout.tsx
app/blog/page.tsx

To the app/blog/layout.tsx does not be nested on app/layout.tsx, but render up isolated, by removing the fist one? or the best approach...

### Additional information

https://nextjs.org/docs/app/building-your-application/routing/layouts-and-templates

**Comment by icyJoseph** [comment]
Do you mean, you need to use Route Groups? https://nextjs.org/docs/app/building-your-application/routing/route-groups

**Reply by izanoth** [reply]
Hi @icyJoseph , thank you for the tip! 
At the first glance, that's it! I will be checking it out
